11-payment-encoding.md

```diff
@@ -90,29 +107,29 @@ field specified below).
 
 ## Rationale
 
-`signature` covers an exact number of bytes because although the SHA-2
-standard actually supports hashing in bit boundaries, it's not widely
-implemented.  The recovery ID allows public key recovery, so the
+`signature` covers an exact number of bytes even though the SHA-2
+standard actually supports hashing in bit boundaries because it's not widely
+implemented. The recovery ID allows public-key recovery, so the
 identity of the payee node can be implied.
 
 ## Tagged Fields
 
-Each Tagged Field is of format:
+Each Tagged Field is of the form:
 
 1. `type` (5 bits)
 1. `data_length` (10 bits, big-endian)
 1. `data` (`data_length` x 5 bits)
 
-Currently defined Tagged Fields are:
+Currently defined tagged fields are:
 
-* `p` (1): `data_length` 52.  256-bit SHA256 payment_hash: preimage of this provides proof of payment.
-* `d` (13): `data_length` variable.  short description of purpose of payment (ASCII),  e.g. '1 cup of coffee'
-* `n` (19): `data_length` 53.  The 33-byte public key of the payee node.
-* `h` (23): `data_length` 52.  256-bit description of purpose of payment (SHA256).  This is used to commit to an associated description which is too long to fit, such as may be contained in a web page.
-* `x` (6): `data_length` variable.  `expiry` time in seconds (big-endian). Default is 3600 (1 hour) if not specified.
+* `p` (1): `data_length` 52. 256-bit SHA256 payment_hash. Preimage of this provides proof of payment
+* `d` (13): `data_length` variable. short description of purpose of payment (ASCII),  e.g. '1 cup of coffee'
+* `n` (19): `data_length` 53. 33-byte public key of the payee node
+* `h` (23): `data_length` 52. 256-bit description of purpose of payment (SHA256). This is used to commit to an associated description that is too long to fit, such as on a web page.
+* `x` (6): `data_length` variable. `expiry` time in seconds (big-endian). Default is 3600 (1 hour) if not specified.
 * `c` (24): `data_length` variable. `min_final_cltv_expiry` to use for the last HTLC in the route. Default is 9 if not specified.
-* `f` (9): `data_length` variable, depending on version. Fallback on-chain address: for bitcoin, this starts with a 5 bit `version`; a witness program or P2PKH or P2SH address.
-* `r` (3): `data_length` variable.  One or more entries containing extra routing information for a private route; there may be more than one `r` field, too.
+* `f` (9): `data_length` variable, depending on version. Fallback on-chain address: for bitcoin, this starts with a 5-bit `version` and contains a witness program or P2PKH or P2SH address.
+* `r` (3): `data_length` variable. one or more entries containing extra routing information for a private route; there may be more than one `r` field
```

Comment:
lower-casing after the period seems weird; is that a thing?

Reply:
No, probably not.